hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -145,6 +148,11 @@ public void init() {
     await().atMost(300, SECONDS).until(this::servicesUp);
     LOG.info(String.format("Waiting for all the containers and services finishes in %d ms",
         System.currentTimeMillis() - currTs));
+    try {
+      Thread.sleep(30000);
```

Comment:
We want to find a better way of checking to see if hive is up, otherwise for now we sleep for 30 sec to make sure things are created.

Reply:
This could be the reason why Azure CI tests take a longer time to finish.  Would be good to figure out a better way to determine whether the environment is ready for tests.